src/cli.js

```diff
@@ -466,5 +466,14 @@ function start (dAuth) {
 }
 
 module.exports = function cli (authDir) {
-  start(auth.getAuth(authDir));
+  if (argv.help) {
+    console.log('Help for Tourbot v0.9.4');
```

Comment:
Can you use package.json here? Should be straight forward with `require()` from package.json.